Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -356,6 +430,13 @@ void EmbeddedFrag::embedFusedRings(const RDKit::VECT_INT_VECT &fusedRings) {
   RDKit::INT_VECT funion;
   RDKit::Union(fusedRings, funion);
 
+  if (useRingTemplates && fusedRings.size() > 1) {
```

Comment:
I'm wondering about the `.size()>1` test here. Couldn't there theoretically also be templates for single (non-fused) rings?

Reply:
Yes! I am planning to do that for macrocycles, but it will be a slightly different templating system. Those templates would be used for individual rings when there is no ring system template. From there I'd do something similar to what coordgen does -- go through possible matches for a macrocycle of a certain size and pick the layout that is best (or close to best) considering other rings and paths that come off the macrocycle.